src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,122 @@
+package nextstep.subway.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static nextstep.subway.acceptance.LineSteps.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 노선 관리 기능")
+class LineAcceptanceTest extends AcceptanceTest {
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철 노선 생성")
+    @Test
+    void createLine() {
+        // when
+        ExtractableResponse<Response> response = 지하철_노선_생성_요청("2호선", "green");
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        ExtractableResponse<Response> listResponse = 지하철_노선_목록_조회_요청();
+
+        assertThat(listResponse.jsonPath().getList("name")).contains("2호선");
+    }
+
+    /**
+     * Given 2개의 지하철 노선을 생성하고
+     * When 지하철 노선 목록을 조회하면
+     * Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+     */
+    @DisplayName("지하철 노선 목록 조회")
+    @Test
+    void getLines() {
+        // given
+        지하철_노선_생성_요청("2호선", "green");
+        지하철_노선_생성_요청("3호선", "orange");
+
+        // when
+        ExtractableResponse<Response> response = 지하철_노선_목록_조회_요청();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertThat(response.jsonPath().getList("name")).contains("2호선", "3호선");
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 조회하면
+     * Then 생성한 지하철 노선의 정보를 응답받을 수 있다.
+     */
+    @DisplayName("지하철 노선 조회")
+    @Test
+    void getLine() {
+        // given
+        ExtractableResponse<Response> createResponse = 지하철_노선_생성_요청("2호선", "green");
```

Comment:
```suggestion
        var 2호선 = 지하철_노선_생성_요청("2호선", "green");
```

인수 테스트인 경우에는 한글 사용하는 것도 가독성에 매우 훌륭하다고 생각해요.

Reply:
한글 반영해보겠습니다.